contentprovider.go

```diff
@@ -319,13 +320,28 @@ func matchScore(secs []DocumentSection, m *LineMatch) float64 {
 			}
 		}
 
+		if f.SymbolInfo != nil {
+			score += scoreKind(language, f.SymbolInfo.Kind)
+		}
+
 		if score > maxScore {
 			maxScore = score
 		}
 	}
 	return maxScore
 }
 
+func scoreKind(language string, kind string) float64 {
+	switch language {
+	case "Java":
```

Comment:
do we need to worry about case? We should have some good regression tests here, since things like changing how we detect languages may change the language string.

Reply:
I added a test cases to e2e_test.go which will fail if our language detector doesn't return "Java" anymore.